src/main/java/uk/gov/hmcts/reform/sscs/service/SendNotificationService.java

```diff
@@ -133,7 +133,7 @@ private void sendLetterNotification(NotificationWrapper wrapper, Subscription su
     }
 
     private void sendMandatoryLetterNotification(NotificationWrapper wrapper, Notification notification, SubscriptionType subscriptionType) {
-        if (hasLetterTemplate(notification) && isMandatoryLetterEventType(wrapper)) {
+        if (isMandatoryLetterEventType(wrapper)) {
```

Comment:
This will cause a NPE if there is no letter template configured for the event type in the application.yaml.